label.js

```diff
@@ -30,12 +31,15 @@ async function label() {
 
   if (ignoreIfAssigned) {
     // check if the issue has been assigned to anyone
-    if (updatedIssueInformation.data.assignees.length != 0) {
+    if (updatedIssueInformation.data.assignees.length !== 0) {
       return "No action being taken. Ignoring because one or more assignees have been added to the issue";
     }
   }
 
   let labels = updatedIssueInformation.data.labels.map(label => label.name);
+  if (ignoreAddIfLabeled && labels.length > 0) {
+    labelsToAdd = [];
```

Comment:
This works, I was thinking that we could move the for loop up into here and its all double negatives. So 👍

Reply:
Yeah, this also helps make sure the log line at the end prints the correct info.